iopipe/contrib/profiler/plugin.py

```diff
@@ -34,7 +34,7 @@ def __init__(self, enabled=False):
 
     @property
     def enabled(self):
-        return self._enabled is True or strtobool(os.getenv('IOPIPE_PROFILER_ENABLED', 'false'))
+        return self._enabled is True or bool(strtobool(os.getenv('IOPIPE_PROFILER_ENABLED', 'false')))
```

Comment:
we have to cast strtobool as a bool?

Reply:
Found an edge case where if you do `strtobool('0')` it returns an `int` and not a `bool`.

Reply:
Interesting, thanks!